src/Init/Util.lean

```diff
@@ -38,6 +38,7 @@ def dbgSleep {α : Type u} (ms : UInt32) (f : Unit → α) : α := f ()
 @[neverExtract, inline] def panicWithPosWithDecl {α : Type u} [Inhabited α] (modName : String) (declName : String) (line col : Nat) (msg : String) : α :=
   panic (mkPanicMessageWithDecl modName declName line col msg)
 
+set_option linter.unusedVariables.funArgs false in
```

Comment:
Mmh, why is this necessary now?

Reply:
I didn't look too deeply into it, but I was getting a warning here and there are similar `set_option` lines elsewhere in the file.

Reply:
I can't reproduce that. In fact, we now compile with `warningAsError`, so the CI should have failed on master if it happened while building Lean.

Reply:
I'll remove it. There shouldn't be any other unrelated changes in this PR, except for the spacing fix and the comment typo. The import reductions may not be the minimum required but I could move those to a separate PR if desired.

Reply:
> I can't reproduce that. In fact, we now compile with `warningAsError`, so the CI should have failed on master if it happened while building Lean.

BTW, I am not sure whether the `warningAsError` is working on the CI. I have found "unused variable warnings" in the last few days, and I believe the CI did not report any problem. Could you please confirm whether this is working or not?